src/main/java/seedu/us/among/logic/endpoint/Request.java

```diff
@@ -55,6 +56,20 @@ public String getAddress() {
      */
     public abstract Response send() throws IOException;
 
+    /**
+     * Creates a http client with timeout set to 60 seconds.
+     *
+     * @return http client to execute request
+     */
+    private CloseableHttpClient createHttpClient() {
+        int timeout = 60;
```

Comment:
Perhaps timeout could be extracted out as a static constant to avoid magic number issues?

Reply:
Great suggestion, changed this in the latest commit! :)